clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -672,6 +672,96 @@ check_function_markers()
         if (!run_checker(memrefs, false))
             return false;
     }
+    // Correctly skip return address check when the return address is
+    // unavailable.
+    {
+        std::vector<memref_t> memrefs = {
+            gen_instr(TID, 1),
+            gen_instr_type(TRACE_TYPE_INSTR_DIRECT_JUMP, TID, CALL_PC, CALL_SZ),
```

Comment:
nit: Kind of odd to have a jump at a pc named CALL_PC and to have its size be CALL_SZ.

Reply:
I've renamed name to JUMP_PC, and JUMP_SZ.